client/a8c-for-agencies/sections/marketplace/hosting-overview-v3/hosting-content/enterprise-agency-hosting/index.tsx

```diff
@@ -1,14 +1,17 @@
-import { VIPLogo, SalesforceLogo, SlackLogo, CNNLogo } from '@automattic/components';
+import { VIPLogo } from '@automattic/components';
 import { Button } from '@wordpress/components';
 import { Icon, external } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
 import { BackgroundType11 } from 'calypso/a8c-for-agencies/components/page-section/backgrounds';
 import SimpleList from 'calypso/a8c-for-agencies/components/simple-list';
 import ProfileAvatar1 from 'calypso/assets/images/a8c-for-agencies/hosting/enterprise-testimonial-1.png';
 import ProfileAvatar2 from 'calypso/assets/images/a8c-for-agencies/hosting/enterprise-testimonial-2.png';
+import CNNLogo from 'calypso/assets/images/logos/cnn.svg';
 import MetaLogo from 'calypso/assets/images/logos/meta.svg';
 import NewYorkPostLogo from 'calypso/assets/images/logos/new-york-post.svg';
 import NewsCorpLogo from 'calypso/assets/images/logos/news-corp.svg';
+import SalesforceLogo from 'calypso/assets/images/logos/salesforce.svg';
+import SlackLogo from 'calypso/assets/images/logos/slack.svg';
```

Comment:
Seems like we have a Slack logo here:

https://github.com/Automattic/wp-calypso/blob/3a11252a2090b9f5113aaf84475becb450361495/client/assets/images/start-with/Slack.svg

Yes, it's white, but we should be able to make it black by specifying `fill: #000` to its `path`.

So, could we possibly use the same svg file in all places?

Reply:
"Same svg file" meaning, literally the same file, or separate files with the same svg code except the fill color?

I briefly touched on the pros/cons [here](https://github.com/Automattic/wp-calypso/pull/99241#discussion_r1940260372), but SVG files loaded in `img` tags cannot have their fill colors changed via CSS. Inlining the SVG will allow CSS styling, but then that is prone to devs forgetting to label them accessibly.

We _could_ set up the logos as inline-able components that allows styling _and_ enforces accessible labeling, but I'm not sure the current usage levels justify the effort at this time. Ultimately I think inline-able components would be the best way though. Could be a good follow-up project at some point.

Reply:
The same as in "not the same file, but a similar one for the same company" that could potentially be reusable instead of having 2 separate copies.

Reply:
Ideally, I'd like to be able to use the same svg everywhere, but I'm not advocating towards needless componentizing. That being said, this is definitely something for a follow up.

Reply:
Agreed. Like a `company.svg` and `company-white.svg` in the same folder. The sizes are all a bit random too, which should ideally be normalized.